molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-debian10/topic_data_creation.yml

```diff
@@ -7,7 +7,8 @@
       shell: |
         kafka-topics --create --topic test-replicator-source \
           --bootstrap-server mds-kafka-broker1:9091 \
-          --command-config /etc/kafka/client.properties
+          --command-config /etc/kafka/client.properties \
+          --replication-factor 1 --partitions 1
       run_once: true
       register: output
       failed_when:
```

Comment:
NIT: but this file is just a duplicate of each other, could move it down a directory and just use relative path to it

Reply:
Done